streams/examples/main/java/org/apache/kafka/streams/examples/pageview/PageViewTypedJob.java

```diff
@@ -118,9 +151,13 @@ public static void main(String[] args) throws Exception {
                 });
 
         // write to the result topic
-        regionCount.to("streams-pageviewstats-output", new JsonPOJOSerializer<>(), new JsonPOJOSerializer<>());
+        regionCount.to("streams-pageviewstats-typed-output", wPageViewByRegionSerializer, regionCountSerializer);
 
         KafkaStreams streams = new KafkaStreams(builder, props);
         streams.start();
+
+        Thread.sleep(8000L);
```

Comment:
We should explain why we sleep here, and (if possible) why it must be 8s / at least 8s.